samples/peripheral/lpuart/src/main.c

```diff
@@ -5,6 +5,8 @@
  */
 
 #include <zephyr.h>
+#include <device.h>
```

Comment:
This will anyway get included through `<drivers/uart.h>`.

Reply:
I prefer to be explicit about header inclusion (we're using device.h APIs)

Reply:
I see. It just seems a bit redundant to me, as when a device driver is used, you by definition have the device API defined.